apps/condo/pages/ticket/[id]/index.tsx

```diff
@@ -106,6 +109,20 @@ export const TicketFileList: React.FC<ITicketFileListProps> = ({ files }) => {
         a.click()
     }
 
+    const handleFileDownload: (file: UploadFile) => void = async (file: UploadFile) => {
+        const forbiddenFileTypes = ['html', 'svg', 'txt']
+        const fileType = file.name.split('.').reverse()[0]
```

Comment:
such Regex would work better `/.+\.(svg|pdf|txt)$/gi`

Reply:
It's probably better to use `/.+\.(svg|pdf|txt)$/gi` rather than `/.*\.(svg|pdf|txt)$/gi` (`*` instead of `+`). I think theoretically the filename can be empty

Reply:
and without flag `g`